bigquery_etl/cli/query.py

```diff
@@ -1227,17 +1227,16 @@ def render(name, sql_dir, output_dir):
     if name is None:
         name = "*.*"
 
-    query_files = paths_matching_name_pattern(name, sql_dir, None)
+    query_files = paths_matching_name_pattern(name, sql_dir, project_id=None)
+    sql_dir = sql_dir.resolve()
     for query_file in query_files:
         rendered_sql = render_template(
             query_file.name, template_folder=query_file.parent, templates_dir=""
         )
 
         if output_dir:
             sql_dir = Path(sql_dir)
```

Comment:
Need to convert it to `Path` first.
```suggestion
    sql_dir = Path(sql_dir).resolve()
    for query_file in query_files:
        rendered_sql = render_template(
            query_file.name, template_folder=query_file.parent, templates_dir=""
        )

        if output_dir:
```
I also sometimes worry about changing passed-in arguments too drastically, so if it were me I might use a separate variable like `resolved_sql_dir`.